src/main/java/com/powsybl/network/conversion/server/dto/EquipmentInfos.java

```diff
@@ -87,22 +87,30 @@ public static Set<VoltageLevel> getVoltageLevels(@NonNull Identifiable<?> identi
 
     // TODO This function need to transfer to poswybl-ws-commons at the new release
     public static String getEquipmentTypeName(@NonNull Identifiable<?> identifiable) {
-        return identifiable.getType() == IdentifiableType.HVDC_LINE ? getHvdcTypeName((HvdcLine) identifiable) : identifiable.getType().name();
+        return switch (identifiable.getType()) {
+            case IdentifiableType.HVDC_LINE -> getHvdcLineTypeName((HvdcLine) identifiable);
+            case IdentifiableType.HVDC_CONVERTER_STATION -> getHvdcConverterStationTypeName((HvdcConverterStation<?>) identifiable);
+            default -> identifiable.getType().name();
+        };
     }
 
     /**
      * @param hvdcLine The hvdc line to get hvdc type name
      * @return The hvdc type name string
      * @throws NetworkConversionException if converter station types don't match
      */
-    private static String getHvdcTypeName(HvdcLine hvdcLine) {
+    private static String getHvdcLineTypeName(HvdcLine hvdcLine) {
         if (hvdcLine.getConverterStation1().getHvdcType() != hvdcLine.getConverterStation2().getHvdcType()) {
             throw NetworkConversionException.createHybridHvdcUnsupported(hvdcLine.getId());
         }
 
         return String.format("%s_%s", hvdcLine.getType().name(), hvdcLine.getConverterStation1().getHvdcType().name());
     }
 
+    private static String getHvdcConverterStationTypeName(HvdcConverterStation<?> hvdcConverterStation) {
+        return String.format("%s_%s", hvdcConverterStation.getHvdcType(), "CONVERTER_STATION");
```

Comment:
why not
```suggestion
        return String.format("%s_%s", hvdcConverterStation.getType().name(), hvdcConverterStation.getHvdcType().name());
```
to have HVDC_CONVERTER_STATION_LCC/HVDC_CONVERTER_STATION_VSC and be more consistent with HVDC lines?
Could at least use `.name()` for consistency

Reply:
Ok probably because it's already used in many places, ok for me